### ISSUE TITLE
1-Click Installers Published for Windows (requires 20 GB GPU - tested on RTX 3090), RunPod and Massed Compute, Amazing Work Congrats

### ISSUE THREAD

**Issue by FurkanGozukara** [Tencent/Hunyuan3D-1#7]
1-Click Installers Published for Windows (requires 20 GB GPU - tested on RTX 3090), RunPod and Massed Compute, Amazing Work Congrats : https://www.patreon.com/posts/115412205

Thank you so much for publishing model and Gradio app

**Installers are shared here : https://www.patreon.com/posts/115412205**

The examples generated with 90000 faces. More faces may improve quality will test hopefully

![20](https://github.com/user-attachments/assets/56d72ff1-2a77-4e73-af51-bb5e662a9a6b)

![19](https://github.com/user-attachments/assets/8b5e81e2-f312-4aa2-9ca6-107dad88f764)

![01](https://github.com/user-attachments/assets/37cb242e-409a-45a4-9f99-d8faabc2cb1c)
![02](https://github.com/user-attachments/assets/f3b6701c-8d3e-4356-af8e-8942e4760926)
![03](https://github.com/user-attachments/assets/b3b53946-f7c8-49ef-8fcb-69459ab0e992)
![004](https://github.com/user-attachments/assets/2272c24e-f611-4250-9160-73e9edd761fd)
![04](https://github.com/user-attachments/assets/8feea3c0-0bb6-4011-9516-87a97a6635f3)
![05](https://github.com/user-attachments/assets/6cde2878-0a55-448a-b0b6-42620fcbd1c4)
![06](https://github.com/user-attachments/assets/e6a08775-d9ef-4f04-8aa9-407d4e46b160)
![07](https://github.com/user-attachments/assets/537c86e7-719a-439d-9896-d9326cc84e40)
![08](https://github.com/user-attachments/assets/38b8dc58-50c5-4d14-81c6-db1fb3250823)
![09](https://github.com/user-attachments/assets/542ae897-2c1b-4f82-b30f-cbcdee81136a)
![10](https://github.com/user-attachments/assets/42a1cbb6-7790-4c8f-97a7-925208ec1f30)
![11](https://github.com/user-attachments/assets/cae39226-cb84-4b65-89a4-5c69170dc508)
![12](https://github.com/user-attachments/assets/e9449eef-612a-40a3-a152-da90fc376f5c)
![13](https://github.com/user-attachments/assets/c1da1a22-70aa-4bd2-9bca-48406d8c048c)
![14](https://github.com/user-attachments/assets/820a9812-bf9d-4bee-aa4b-5d4d0fb50f3c)
![15](https://github.com/user-attachments/assets/3830fa34-09e4-455c-8b8d-57f11fe4d23c)
![16](https://github.com/user-attachments/assets/ce8995ae-6e69-41b4-999f-7bbb402031a4)
![17](https://github.com/user-attachments/assets/1f7f7a14-7c77-4c69-922e-1af76d893a69)

**Comment by SoftologyPro** [comment]
Paywalled spam.  Should be deleted.

**Comment by FurkanGozukara** [comment]
> Paywalled spam. Should be deleted.

how about you make and publish and make more people to know and make them use?

my work only helps the researchers and average people

**Comment by SoftologyPro** [comment]
> > Paywalled spam. Should be deleted.
> 
> how about you make and publish and make more people to know and make them use?
> 
> my work only helps the researchers and average people

I already do.  See Visions of Chaos.  Hundreds of AI systems for free with 1 click installers.  Hunyuan#d-1 will be included in the next release.  Any Windows users interested in running AI systems locally should check it out.

Can you help with a WHL for Pytorch3D for Windows?

**Comment by FurkanGozukara** [comment]
> > > Paywalled spam. Should be deleted.
> > 
> > 
> > how about you make and publish and make more people to know and make them use?
> > my work only helps the researchers and average people
> 
> I already do. See Visions of Chaos. Hundreds of AI systems for free with 1 click installers. Hunyuan3d-1 will be included in the next release. Any Windows users interested in running AI systems locally should check out https://softology.pro/voc.htm
> 
> Can you help with a WHL for Pytorch3D for Windows?

good then you can open your thread

i would never come and attack your post hope that helps

**Comment by SoftologyPro** [comment]
I see you already opened a issue for a Windows WHL :)  Can you share your WHL if you managed to build one?

The only reason I said it should be deleted is this is spam.  Github is not for taking an open/free repo and charging people for an installer script.  Surely you understand that.

**Comment by FurkanGozukara** [comment]
> I see you already opened a issue for a Windows WHL :) Can you share your WHL if you managed to build one?
> 
> The only reason I said it should be deleted is this is spam. Github is not for taking an open/free repo and charging people for an installer script. Surely you understand that.

yes i solved (spent over 7 full hours worktime to make these installers work) and surely this isnt spam. i am the biggest promoter of the Hunyuan3D-1 right now.  and good luck to you

just 2 posts over 50k views and i share on my all platforms

![image](https://github.com/user-attachments/assets/ff948542-b343-47f0-bd80-cce91e9ba21f)

![image](https://github.com/user-attachments/assets/10cb872d-49d1-4316-a8dd-c1047828b003)

**Comment by SoftologyPro** [comment]
OK, never mind, I built my own WHL and have a 1 click installer working for the next release of Visions of Chaos.

And remember everyone, Visions of Chaos is free and includes this and hundreds of other AI and machine learning systems for you to run locally under Windows.  No need to pay for a paywalled install script for this one system.

See how annoying spam ads can be in git issues? :)

**Comment by matbee-eth** [comment]
> > Paywalled spam. Should be deleted.
> 
> how about you make and publish and make more people to know and make them use?
> 
> my work only helps the researchers and average people

Two things can be true:

- Annoying spammer
- Giving researchers reach

You're both.

**Comment by Mingsirdon** [comment]
It's you again, the person making money from open-source software
